### PR TITLE
Fix ffq dependency spec

### PR DESCRIPTION
**Pull request by bentsherman**
Figured out how to specify the dependency correctly in `setup.cfg`. Also changed to use Phil's commit instead of `devel` branch.

### CODE REVIEW

**Review by ewels — APPROVED** [review]

File: setup.cfg [hunk 1]
```diff
@@ -31,7 +31,7 @@ install_requires =
     fastapi>=0.68.0
     pydantic>=1.8.0
     uvicorn>=0.15.0
-    git+https://github.com/pachterlab/ffq@devel
+    ffq @ git+https://github.com/pachterlab/ffq@1d77aeaa
```
Comment (ewels) [inline setup.cfg:34]: I guess this is the `setup.cfg` equivalent of doing `#egg=ffq` in `setup.py`?

Comment (bentsherman) [reply setup.cfg:34]: I guess so. I didn't even realize it was different from running `pip install` directly or using a requirements file. I just assumed pip would infer everything from the git repo.

Comment (ewels) [reply setup.cfg:34]: Python packaging is a bit of a mess for this stuff 😞

Comment (pditommaso) [reply setup.cfg:34]: Should I merge it?

Comment (bentsherman) [reply setup.cfg:34]: yes. I tested more thoroughly this time.

Comment (pditommaso) [reply setup.cfg:34]: vamos!

**Comment by pditommaso** [comment]
Ok, it's avail here https://ffq.staging-tower.xyz

**Pull request merged**